region/tourian/main/Metroid Room 3.json

```diff
@@ -205,22 +224,25 @@
       "requires": [
         {"enemyKill": {
           "enemies": [["Metroid", "Metroid", "Metroid"]],
-          "excludedWeapons": ["Super", "Missile"]
+          "explicitWeapons": ["PowerBombPeriphery"]
         }},
-        {"metroidFrames": 272}
+        {"metroidFrames": 300}
       ],
       "clearsObstacles": ["A"]
     },
     {
       "link": [1, 2],
-      "name": "Metroid Room 3 PB Dodge Kill (Left to Right)",
-      "notable": true,
+      "name": "3 PB Dodge Kill (Left to Right)",
       "requires": [
         {"enemyKill": {
           "enemies": [["Metroid", "Metroid", "Metroid"]],
-          "excludedWeapons": ["Super", "Missile"]
+          "explicitWeapons": ["PowerBomb"]
         }},
-        "canMetroidAvoid"
+        "canMetroidAvoid",
+        {"or": [
+          {"metroidFrames": 120},
+          "canInsaneJump"
+        ]}
       ],
       "clearsObstacles": ["A"],
       "note": [
```

Comment:
possibly [1, 2] and [2, 1] `Harder Evade` should also require `Morph` or `canTrickyJump`